client/src/components/About/AboutInformation/AboutInformation.js

```diff
@@ -27,13 +27,20 @@ export default class AboutInformation extends React.Component {
         const { contributors, monumentStatistics } = this.props;
         const { showingAllContributors } = this.state;
 
-        let mapNavLink = (
+        const mapNavLink = (
             <NavLink onClick={e => {
                 e.preventDefault();
                 window.location.replace('/map');
             }} to="/map" key="map">click here</NavLink>
         );
 
+        const nineElevenMemorialLink = (
+            <NavLink onClick={e => {
+                e.preventDefault();
+                window.location.replace('/monuments/2732');
+            }} to="/monuments/2732" key="911-memorial">9/11 Memorial</NavLink>
```

Comment:
Should we be doing the `NavLink` to a particular Monument like this? It won't work locally but it's the easy way to do it. Otherwise we'd have to search for it and thee's no guarantee the search would return what we want. Thoughts?

Reply:
I don't think anticipate the ID changing on production but who knows.

Reply:
This feels very unsafe. I think when we load all the stats we could include this in the payload by doing a search on the backend for the exact name and approximate location. If somehow there's 2 results just take the first one. If there's no results (for example locally) just don't link to anything I guess

Reply:
Sounds good. I'll make that change now.

Reply:
Done.